### ISSUE TITLE
Clicking the Filesystem Folder under 3 dot menu results in a 401 Unauthorized web page and a Failed Login Notification

### ISSUE THREAD

**Issue by jweston2112** [music-assistant/hass-music-assistant#765]
### What version of Music Assistant has the issue?

2022.8 beta 0

### The problem

I click on the File Icon labeled Filesystem under the 3 dot menu and a new tab opens in the browser (Edge and Chrome)
and it says 401 Unauthorized , and I get a Notification that states failed login from the IP address I was using Music Assistant

### How to reproduce

Choose Artist /Album/Track and Click the Icon 

### Relevant log output

```
Logger: homeassistant.components.http.ban
Source: components/http/ban.py:84
Integration: HTTP ([documentation](https://rc.home-assistant.io/integrations/http), [issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+http%22))
First occurred: 1:48:58 PM (2 occurrences)
Last logged: 1:53:46 PM

Login attempt or request with invalid authentication from 192.168.128.2 (192.168.128.2). Requested URL: '/media/MUSIC/Styx'. (Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.5060.134 Safari/537.36 Edg/103.0.1264.77)
Login attempt or request with invalid authentication from 192.168.128.2 (192.168.128.2). Requested URL: '/media/MUSIC/Various%20Artists/Greatest%20Hits'. (Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36)

```

### What version of Home Assistant Core are your running

2022.8.0b2

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Raspberry Pi

**Comment by marcelveldt** [comment]
That link should be disabled for the filesystem provider... Looks like I missed a spot.
It will only work for http locations

**Comment by jweston2112** [comment]
Well I can tell you ... if you click it too many times and you have IP Ban setup.... it does... :)

**Comment by marcelveldt** [comment]
haha, oops. I'll fix it. Maybe make it a tiny bit smarter and link to the "browse" section instead... 
For streaming providers that link takes you to the details page of the item in their catalog but for the file provider it tries opening a file location which it can't

**Comment by jweston2112** [comment]
Well  Found it in 2022.8 beta 0... but I just checked my Production system and its in 2022.7.4 as well... just a FYI

**Comment by OzGav** [comment]
Fixed in MA V2.0.0b12
